server/errors.go

```diff
@@ -92,4 +92,5 @@ var (
 	errHTTPInternalError                             = &errHTTP{50001, http.StatusInternalServerError, "internal server error", ""}
 	errHTTPInternalErrorInvalidPath                  = &errHTTP{50002, http.StatusInternalServerError, "internal server error: invalid path", ""}
 	errHTTPInternalErrorMissingBaseURL               = &errHTTP{50003, http.StatusInternalServerError, "internal server error: base-url must be be configured for this feature", "https://ntfy.sh/docs/config/"}
+	errHTTPWontStoreMessage                          = &errHTTP{50701, http.StatusInsufficientStorage, "topic is inactive; no device available to recieve message", ""}
```

Comment:
I need to add 507 to the UP spec and discuss it with the group further. For now, this can stay.

Reply:
"Insuffiencient storage" implies that it's stored on the server, and a "5xx" error implies that it's the server's fault that the request was denied. Also, 507 seems WebDAV specific.

Since there are two clients involved (sender (= traditional client) ---> ntfy server --> subscriber), it's hard to find a good HTTP code, but I'd still say it's a 4xx, no?

Maybe a 409?

> 409 Conflict - Indicates that the request could not be processed because of conflict in the current state of the resource, such as an [edit conflict](https://en.wikipedia.org/wiki/Edit_conflict) between multiple simultaneous updates.

Or a 412? Probably less so..

> 412 Precondition Failed - The server does not meet one of the preconditions that the requester put on the request header fields.

Or 422?

> 422 Unprocessable Entity - The request was well-formed but was unable to be followed due to semantic errors.

I think 409 and 422 work well.

Reply:
IMHO by not it's not a 4xx since there's nothing the app server can do about this. It's the push server's "fault" (design limitation) that it doesn't know whether the subscription is valid or not.

Regardless, let's discuss this further in https://github.com/UnifiedPush/specifications/issues/34

Reply:
I'll make it 409 here for now. This should be trivial to update once the spec is ready. Also, really sorry for the delay, I'll try to implement all the comments by noon-ish today.

Reply:
No worries dude. It's all my fault. I didn't look at your PR for ages and now I'm rushing you. It's unfair ... I can help with the code and/or tests if you like. Maybe I can write some tests already, TDD style :-)

Reply:
Tests would absolutely help!